cosmic-applet-network/i18n/nl/cosmic_applet_network.ftl

```diff
@@ -1,18 +1,20 @@
 network = Netwerk
-airplane-mode = Vliegtuig modus
-airplane-mode-on = Vliegtuig modus staat aan
-turn-off-airplane-mode = Zet aan om Wi-Fi, Bluetooth en Telefonie te activeren.
```

Comment:
Line 4: "mobile broadband" doesn't translate to "Telefonie": (Cellphone).
"mobile broadband" -> "mobile data"